pkg/audit/manager.go

```diff
@@ -480,6 +480,7 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		gvk := r.Constraint.GroupVersionKind()
 		enforcementAction := r.EnforcementAction
 		message := r.Msg
+		details := r.Metadata["details"]
```

Comment:
Just pointing out for reviewers that `details` is always at least set to `{}` here: https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/client/regolib/src.go#L15